datadog_checks_base/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG - datadog_checks_base
 
+## 23.7.2 / 2022-01-08
```

Comment:
Can you remove this change from this release? The check has already been released from the [7.33.x release branch](https://github.com/DataDog/integrations-core/pull/11072) and it could cause an issue with the pipeline if it's released again. Same with the http_check. I can update the changelog manually afterward.